app/(app)/tickets/index.tsx

```diff
@@ -129,28 +128,34 @@ const TicketsRoute = ({ active }: RouteProps) => {
 
   if (active && !tickets?.length) {
     return (
-      <EmptyStateScreen
-        contentTitle={t('noActiveTickets')}
-        text={t('noActiveTicketsText')}
-        actionButton={
-          <Link href="/purchase" asChild>
-            <Button variant="primary">{t('buyTicket')}</Button>
-          </Link>
-        }
-      />
+      <View className="flex-1 justify-center">
+        <ContentWithAvatar
+          title={t('noActiveTickets')}
+          text={t('noActiveTicketsText')}
+          customAvatarComponent={<EmptyStateAvatar />}
+          actionButton={
+            <Link href="/purchase" asChild>
+              <Button variant="primary">{t('buyTicket')}</Button>
+            </Link>
+          }
+        />
+      </View>
     )
   }
   if (!active && !tickets?.length) {
     return (
-      <EmptyStateScreen
-        contentTitle={t('noHistoryTickets')}
-        text={t('noHistoryTicketsText')}
-        actionButton={
-          <Button variant="primary" onPress={handleFiltersPress}>
-            {t('filters')}
-          </Button>
-        }
-      />
+      <View className="flex-1 justify-center">
```

Comment:
We have ScreenViewCentered, is it the same functionality?

Reply:
I cannot add ScreenViewCentered, because it's already wrapped with one ScreenView (in parent component)